lib/pf/UnifiedApi.pm

```diff
@@ -107,6 +105,10 @@ our @API_V1_ROUTES = (
 
     { controller => 'nodes', collection_v2a => { get => 'latest_locationlog_by_mac' }, resource_v2a => {}, path => '/nodes/:mac/latest_locationlog' },
     { controller => 'nodes', collection_v2a => { get => 'locationlog_by_mac' }, resource_v2a => {}, path => '/nodes/:mac/locationlog' },
+
+    { controller => 'violations', resource_v2a => {}, collection_v2a => { get => 'read_list', post => 'create' }, path => '/violations' },
+    { controller => 'violations', resource_v2a => {}, collection_v2a => { get => 'read_list_by_search' }, path => '/violations/:search' },
+    { controller => 'violations', resource_v2a => {}, collection_v2a => { get => 'read_row_by_id' }, path => '/violation/:id' },
```

Comment:
Based on my other comment, this doesn't have much use since the CRUD controller will already handle reading a row by its id